NDB.Covid19/NDB.Covid19.iOS/Views/ConsentView/ConsentViewController.cs

```diff
@@ -180,7 +180,7 @@ void GoToResultPage()
             });
         }
 
-        public void SetWarningViewVisibility() { WarningView.Alpha = _consentViewModel.ConsentIsGiven ? 0 : 100; }
+        public void SetWarningViewVisibility() { WarningView.Hidden = _consentViewModel.ConsentIsGiven ? true : false; }
```

Comment:
public void SetWarningViewVisibility() { WarningView.Hidden = _consentViewModel.ConsentIsGiven } 

for extra developer XP :)